packages/react/src/avatar/Avatar.tsx

```diff
@@ -54,10 +52,18 @@ export const Avatar = forwardRef<HTMLSpanElement, AvatarProps>(
           <Box asChild objectFit="cover" rounded="inherit" size="full">
             <RadixAvatar.Image alt={name} src={src} />
           </Box>
-          <Box asChild {...styles.fallback({ size })}>
+          <Box asChild {...styles.fallback({})}>
             <RadixAvatar.Fallback delayMs={FALLBACK_DELAY_IN_MS}>
               {/* TODO: Add a generic user icon, if `children` is `undefined` */}
-              {icon ? icon : name ? getInitialsFromName(name) : children}
+              {icon ? (
```

Comment:
let's revert these changes from this PR? since the other avatar PR already has them - this PR should only have avatar group related changes

Reply:
Vai, I rebased the branch with avatar design fix, as I needed the updated style. Do I not do it?

Reply:
oops didn't check the commits - ignore this comment then